index.html

```diff
@@ -31,9 +31,11 @@ <h3>Next meeting</h3>
       <p>
         <b>Topics:</b>
         <p>
-            <a href="https://github.com/BeastyBlacksmith">Simon</a> will explain how bacteria find their center and shows how to match experiments with julia.
+          <a href="https://github.com/j-fu">Jürgen</a> will give an introduction to the solution of
+          partial differential equations using the finite volume method and discuss his package
+          <a href="https://github.com/j-fu/VoronoiFVM.jl">VoronoiFVM.jl</a>.
         </p>
-        <b>Date/time:</b> Tuesday, September 10, 2019, 18:30</br>
+        <b>Date/time:</b> Tuesday, October 9, 2019, 18:30</br>
```

Comment:
Shoul be *October 8*, I guess.

Reply:
Indeed... Please feel free to change this..

Jürgen